integrations/facebook-pixel/test/index.test.js

```diff
@@ -1992,6 +2090,30 @@ describe('Facebook Pixel', function() {
 
         analytics.called(window.fbq, 'trackSingle', options.pixelId, '123456');
       });
+
+      it('should send facebook pixel properties for product search', function() {
+        analytics.track('Products Searched', {
+          query: 'yo',
+          content_category: 'Cars',
```

Comment:
`content_category`, `content_ids`, and `contents` are not Segment-specced properties, so shouldn't appear in a `track` call.